code/datums/mood_events/drink_events.dm

```diff
@@ -26,3 +26,13 @@
 	description = "<span class='nicegreen'>Amazing taste!</span>\n"
 	mood_change = 50
 	timeout = 10 MINUTES
+	
+/datum/mood_event/full_on_pilk
+	description = "<span class='nicegreen'>I am now full on pilk! That was some amazing bubbly goodness!</span>\n"
+	mood_change = 7
+	timeout = 7 MINUTES
+
+/datum/mood_event/pegged
+	description = "<span class='nicegreen'>OH YEAH, NOW IM PEGGED!</span>\n"
```

Comment:
:face_with_raised_eyebrow:

Reply:
![image](https://user-images.githubusercontent.com/75863639/142679371-ce23b06b-5904-4ece-a976-3e06f171093d.png)
___
![image](https://user-images.githubusercontent.com/75863639/142679445-0857d7ce-8a25-4549-9654-26b9ee0a1fae.png)

Reply:
I had the same reaction when I saw this too

And I fucking hate it 100%